example/minio_standalone/main.tf

```
@@ -22,7 +22,10 @@ module "minio" {
   container_environment_variables = ["SOME_VAR_N1=some-value"]
   use_host_volume                 = true
   use_canary                      = true
-
+resource_proxy = {
+  cpu = 200
+  memory = 128
+}
```

Should be indented. Not suggesting here, not quite sure how many spaces the indent is :sweat_smile:
```suggestion
resource_proxy = {
  cpu = 200
  memory = 128
}
```